src/components/Data/Documents/Update.vue

```diff
@@ -83,17 +105,17 @@
         documentToEditId
       }
     },
+    data () {
+      return {
+        show: false
+      }
+    },
     ready () {
+      this.fetch()
       kuzzle
         .dataCollectionFactory(this.collection, this.index)
-        .fetchDocumentPromise(this.documentToEditId)
-        .then(res => {
-          this.setNewDocument(res.content)
-          this.$broadcast('document-create::fill', res.content)
-          return null
-        })
-        .catch(err => {
-          this.$dispatch('toast', err.message, 'error')
+        .subscribe({term: {_id: this.documentToEditId}}, () => {
```

Comment:
You probably need to store the subscription and perform `room.unsubscribe()` on `beforeDestroy()`hook.

```
let subscription
[...]
export default {
[...]
ready() {
  this.fetch()
  subscription = kuzzle
    .dataCollectionFactory(this.collection, this.index)
    .subscribe([...])
},
beforeDestroy () {
  subscription.unsubscribe()
}
```